internal/support/notifications/rest_subscription.go

```diff
@@ -94,6 +101,13 @@ func subscriptionHandler(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
+		// validate if email addresses contain CRLF pair
+		if err = validateEmailAddresses(s); err != nil {
+			http.Error(w, err.Error(), http.StatusBadRequest)
```

Comment:
Do we need do any type checking here as well(See my previous comment)?

Reply:
Yes, both are identical in nature. Done.